examples/resnet50_server_pytorch.py

```diff
@@ -23,6 +26,7 @@
 INFERENCE_BATCH_SIZE = 16
 
 
+# define the input and output schemas for validation
 class ImageReq(BaseModel):
     image: str  # base64 encoded
```

Comment:
I feel it's not a good example to demonstrate how to use `pydantic`. Only one string is required here. Maybe we can provide some simple examples that using `msgpack`, `pydantic`, `protobuf` properly.

Reply:
You mean more complex data structure? Different fields with various types? How about adding fields like "request_id", "img_size"?

Reply:
Yes. Maybe we can use `pydantic` in the XGBoost example, since it contains lots of parameters for inference.

Reply:
Ok, then I remove it from here for now.